test/caching-precompiler.js

```diff
@@ -77,8 +77,8 @@ test('should reuse existing source maps', t => {
 	const tempDir = uniqueTempDir();
 	const precompiler = new CachingPrecompiler({path: tempDir, getBabelOptions, babelCacheKeys});
 
-	precompiler.precompileFile(fixture('es2015-source-maps.js'));
-	const options = transformSpy.lastCall.args[1];
+	precompiler.precompileFile(fixture('es2015-source-maps.js'), []);
+	const options = babel.transform.lastCall.args[1];
```

Comment:
I suspect you may have had a merge conflict here? Use `transformSpy` rather than `babel.transform` in this line.

Reply:
Oh alright gotcha